package.json

```diff
@@ -5,14 +5,6 @@
   "main": "./lib/cjs/index.js",
   "module": "./lib/esm/index.js",
   "types": "./lib/types/index.d.ts",
-  "exports": {
-    ".": {
-      "import": "./lib/esm/index.js",
-      "require": "./lib/cjs/index.js"
-    },
-    "./esm": "bundles/esm.bundle.js",
-    "./web": "bundles/web.bundle.js"
```

Comment:
will this not break the https://github.com/warp-contracts/warp/issues/152#issuecomment-1142054586 ?

Reply:
To reference the bundled web and esm you would use 

```js

import { WarpWebFactory } from 'warp-contracts/bundles/esm.bundle'

```

instead of 

```js

import { WarpWebFactory } from 'warp-contracts/esm'

```

Reply:
FWIW, I continue to struggle with this issue, but I have narrowed down the root cause. The `./lib/esm/index.js` build is not esm compliant, I don't know why it is not, but I do know when I switch the exports config to point the `import` to cjs it works fine.

```json
"exports": {
    ".": {
      "import": "./lib/cjs/index.js",
      "require": "./lib/cjs/index.js"
    },
    "./esm": "bundles/esm.bundle.js",
    "./web": "bundles/web.bundle.js"
  },
```

So that may be another option. 

The current esm build results in this response:

```
file:///workspace/stamp-coin/copy-and-deploy.mjs:4
import { WarpFactory } from './warp/lib/esm/index.js'
         ^^^^^^^^^^^
SyntaxError: Named export 'WarpFactory' not found. The requested module './warp/lib/esm/index.js' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from './warp/lib/esm/index.js';
const { WarpFactory } = pkg;
```

For now, I am just pulling down the latest source and patching the package.json

```
npx degit warp-contracts/warp#v1.2.0-rc.1 warp
cd warp
npx json -I -f package.json -e 'this.exports.import = "./lib/cjs/index.js"'
yarn && yarn build:cjs
```

Please, let me know how I can help move this bug to fixed.